index.html

```diff
@@ -66,35 +66,35 @@ <h2 class="lead secondary-heading">
     <div class="row">
       <!-- 1 -->
       <div class="col-lg-4 col-md-6 feature">
-        <h3>Julia is fast!</h3>
+        <h3>Fast</h3>
         <p>Julia was designed from the beginning for <a href="/benchmarks/">high performance</a>.
         Julia programs compile to efficient native code for <a href="/downloads/#support_tiers">multiple platforms</a> via LLVM.</p>
       </div>
       <!-- 2 -->
       <div class="col-lg-4 col-md-6 feature">
         <h3>Dynamic</h3>
         <p>Julia is <a href="https://docs.julialang.org/en/v1/manual/types/">dynamically typed</a>, feels like a scripting language, and has good support for <a href="https://docs.julialang.org/en/v1/stdlib/REPL/">interactive</a> use.</p>
+</p>
       </div>
       <!-- 3 -->
       <div class="col-lg-4 col-md-6 feature">
-        <h3>Optionally typed</h3>
-        <p>Julia has a <a href="https://docs.julialang.org/en/v1/manual/types/">rich language of descriptive datatypes</a>, and type declarations can be used to clarify and solidify programs.</p>
+        <h3>Reproducible</h3>
+        <p><a href="https://julialang.github.io/Pkg.jl/v1/environments/">Immutable environments</a> let you recreate the same Julia environment every time, across platforms, with <a href="https://binarybuilder.org">pre-built binaries</a>.</p>
```

Comment:
I wonder if "immutable environments" is the right term here. You can modify an environment (add/remove dependencies to it). Maybe "environments and immutable package versions"? Or "manifests"? It's also a bit circular (environments let you recreate environments). Maybe "Environments let you recreate an identical software stack every time, ..."? cc @StefanKarpinski

Reply:
Coming to think about it, for this position - I don't like having the word "immutable" at all. Maybe something simpler?

Reply:
"reproducible"?

Reply:
I was also thinking that, but that is the also the header. Maybe that is OK though.

Reply:
I think using the same words is good to reinforce the point.